src/App.js

```diff
@@ -34,6 +35,7 @@ const App = () => (
           <Route exact path="/dashboard/concepts" component={Authenticate(ConceptSearch)} />
           <Route exact path="/dashboard/dictionaries" component={Authenticate(DictionaryDisplay)} />
           <Route exact path="/dashboard/userdictionaries" component={Authenticate(OwnerDictionary)} />
+          <Route exact path="/dashboard/bulkconcepts" component={Authenticate(AddBulkConcepts)} />
```

Comment:
The route shouldn't have anything to do with `dashboard` `concept` would be more appropriate

Reply:
ok let me work on that